### DISCUSSION TITLE
Is it possible to make ElementMaterialSampler to sample value based on the centroid points of element instead of quadrature points

### DISCUSSION THREAD

**Discussion by Yulongggggg** [idaholab/moose#29122] [Q&A Modules: Solid mechanics]
Is it possible to make ElementMaterialSampler to sample value based on the centroid points of element instead of quadrature points

**Comment by GiudGiud** [comment]
Hello

You can certainly modify it to make it so. 
But currently in its present state, no

Guillaume
